lib/usd/translators/shading/usdFileTextureWriter.cpp

```diff
@@ -221,6 +223,11 @@ PxrUsdTranslators_FileTextureWriter::Write(const UsdTimeCode& usdTime)
         return;
     }
 
+    MString colorRuleCmd;
+    colorRuleCmd.format(
+        "colorManagementFileRules -evaluate \"^1s\";", fileTextureNamePlug.asString());
+    const MString colorSpaceByRule = MGlobal::executeCommandStringResult(colorRuleCmd);
+
```

Comment:
Maybe move this closer to where we get the `colorSpace` plug, or even inside that `if (status == MS::kSuccess) {` block?